tables/ru-litbrl.ctb

```diff
@@ -733,7 +741,18 @@ nofor context #5=1@2456 "w"
 nofor context #5=1@1346 "x"
 nofor context #5=1@13456 "y"
 nofor context #5=1@1356 "z"
-nofor context #5=1[!%lowerlatin] *#5=0
+nofor context #5=1@12356 "\x00e0"
```

Comment:
@Futyn-Maker Just noticed that for these back-translation fixes there are no test changes.